package.json

```diff
@@ -66,8 +66,8 @@
     "npm-run-all": "4.1.5",
     "postcss": "8.4.25",
     "prettier": "2.8.8",
-    "react-test-renderer": "^18.2.0",
     "prisma": "4.16.2",
+    "react-test-renderer": "^18.2.0",
```

Comment:
@GuillermoFloresV  asked @ngillux  
> Were there any issues with students not being able to use it leading to the changes in package.json?

The package.json change is just an alphabetical sorting that `npm` and `npx` do automatically.